src/features/extract.js

```diff
@@ -52,9 +52,7 @@ if (isMainThread) {
     const clientid = process.argv[2];
 
     if (!clientid) {
```

Comment:
just FYI, extract is no longer maintained at this point, we should move it to a deprecated folder, or something along those lines.

Reply:
Good point, I can do that in an upcoming PR